init.lua

```diff
@@ -599,3 +599,43 @@ minetest.register_chatcommand("verbana_gui", {
         verbana_gui_form:show(player)
     end,
 })
+
+if minetest.global_exists("sway") then
+    local pagename = "verbana_gui"
+    sway.register_page(pagename .. ":gui", {
+        title = S('Verbana'),
+        is_in_nav = function(_, player, _)
+            return verbana.privs.is_moderator(player:get_player_name())
+        end,
+        get = function(_, player, _)
+            return sway.Form {
+                verbana_gui_form:embed {
+                    player = player,
+                    name = pagename,
+                }
+            }
+        end
+    })
+    local function on_priv_change(player_name, _, priv)
+        if not sway.enabled then
+            return
+        end
+
+        if priv ~= verbana.privs.moderator then
+            return
+        end
+
+        local player = minetest.get_player_by_name(player_name)
+        if not player then
+            return
+        end
+
+        if sway.get_page(player) ~= pagename then
```

Comment:
Just re-read this. This needs to be `pagename..":gui"` to match how it's registered in sway.